int32 id 

geometry_msgs/Point32 top_left
geometry_msgs/Point32 right_bottom
